src/store.js

```diff
@@ -1,6 +1,104 @@
+import { fetchBreeds } from "./data.js";
+
+const state = {
+  breedsList: [],
+  searchList: [],
+  isLoading: false,
+  error: "",
+  searchQuery: "",
+  message: "",
+  isInputDisabled: false,
+};
+const mutations = {
+  startLoading() {

```

Comment:
start in stop loading lahko združiš v eno `setLoading` in notr passaš true / false, je bolj pregledno pomoje 👍